app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: 'author_id'
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def recent_posts(limit)
+    posts.order(created_at: :desc).limit(limit)
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=Users-,A%20method%20that%20returns%20the%203%20most%20recent%20posts%20for%20a%20given%20user.,-Posts), Please make the limit 3 posts only.
[hint]: It will be like that:
```ruby
def recent_posts
    posts.order(created_at: :desc).limit(3)
end
```